sdcm/sct_runner.py

```diff
@@ -600,8 +602,67 @@ def _create_instance(self,
         LOGGER.info("Got public IP: %s", instance.public_ip_address)
         self.instance = instance
 
+        # If the SCT runner instance is used as a Docker backend for tests
+        if os.getenv("SCT_CLUSTER_BACKEND") == "docker":
+            LOGGER.info("Configure unused disks to use as a persistent storage for Docker.")
+            LOGGER.info("Connecting to instance...")
```

Comment:
Such kind of a message looks like a `LOGGER.debug(...)`, but not `info`.

Reply:
Changed to debug instead of the very first message where we notify about 'configuring'.
It helps to see immediately in that this step was executed, e.g. in Jenkins, without the need to download and look through test logs